tests/_utils/turnOffErrors.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { Awaitable } from '@ckeditor/ckeditor5-integrations-common';
+import { timeout } from './timeout.js';
+
+export async function turnOffErrors( callback: () => Awaitable<void> ): Promise<void> {
+	const handler = ( evt: ErrorEvent ) => {
+		evt.preventDefault();
+	};
+
+	window.addEventListener( 'error', handler, { capture: true, once: true } );
```

Comment:
To my understanding, this mutes Vitest errors. Would it be possible to use [`expect().toThrowError()`](https://vitest.dev/api/expect.html#tothrowerror) instead?

Reply:
It's not working, I tested it